test/rgb.collection.js

```diff
@@ -291,21 +291,36 @@ exports["RGB.Collection"] = {
 
 
   "Animation.render": function(test) {
-    test.expect(1);
+    test.expect(4);
 
     var rgbs = new RGB.Collection([
       [1, 2, 3],
       [4, 5, 6],
     ]);
 
-    this.color = this.sandbox.stub(RGB.prototype, "color");
+    rgbs.each(function(rgb) {
+      this.sandbox.stub(rgb, "write");
```

Comment:
I had to update these tests that stub `color` to test animations, since the Animation.render method no longer calls `color`.  I can't stub `update` because it's a read-only property, so I'm stubbing `write` which is another layer down, and a better representation of the values actually being sent to the device.